src/slack/slack.js

```diff
@@ -706,28 +728,14 @@ export class Slack {
 
     // office-is_personal-?-note
     if (actionName === 'note') {
-      if (actionValue === 'note-yes') {
-        order.step = 'note_yes'
+      if (actionValue === 'note_yes') {
         order.messages = MESSAGES.office.personal.note
```

Comment:
seeing this line - maybe I forgot to change it to copy the messages, or there was some conflict. anyway, please change it to `[...MESSAGES.office.personal.note]`, thanks